accepted/2021/package-validation/package-validation.md

```diff
@@ -0,0 +1,190 @@
+# Package Validation
+
+**Owner** [Immo Landwerth](https://github.com/terrajobst)
+
+With .NET Core & Xamarin we have made cross-platform a mainstream requirement
+for library authors. However, we lack validation to ensure that library
+developers, which can result in packages that don't work well which in turn
+hurts our ecosystem. This is especially problematic for emerging platforms where
+adoption isn't high enough to warrant special attention by library authors.
+
+The tooling we provide as part of the SDK has close to zero validation that
+multi-targeted packages are well-formed. For example, a package that
+multi-targets for .NET Core 3.1 and .NET Standard 2.0 needs to ensure that code
+compiled against the .NET Standard 2.0 binary can run against the binary that is
+produced for .NET Core 3.1. In the dotnet/runtime repo we have tooling that
+ensures that this is the case, but this can't easily be used by our customers.
+We have seen this issue in the wild, even with 1st parties, for example, the
+Azure AD libraries.
+
+Another common issue in the .NET ecosystem is that binary breaking changes
+aren't well understood. Developers tend to think of "does the source still
+compile" as the bar to determine whether an API is compatible. However, at the
+ABI level certain changes that are fine in C# aren't compatible at the level of
+IL, for example, adding a defaulted parameter or changing the value of a
+constant. In the dotnet/runtime repo we have tooling at allows to validate that
+our current API shape is backwards compatible with the API we shipped before.
+
+This document outlines a set of features that will allow library developers to
+validate that their packages are well-formed and they they didn't make
+unintentional breaking changes. This is achieved by productizing our internal
+tooling that we're planning to use ourselves as well.
+
+## Scenarios and User Experience
```

Comment:
For clarity, am I understanding correctly that this tool runs on exactly one single package at a time? 
Eg., it doesn't need to check referenced packages; nor does it attempt to compare two versions of the same package?

Reply:
Oh never mind I see the package-to-package scenario. Still it would be good to note whether it ever looks at any other packages other than the 1 or 2 supplied

Reply:
> does it attempt to compare two versions of the same package

it will have the functionality to compare with a previous version of the package.

> it doesn't need to check referenced packages;

we will run few checks on the package dependencies.